### ISSUE TITLE
Echoflow package dependencies

### ISSUE THREAD

**Issue by Sohambutala** [OSOceanAcoustics/echoflow#4]
While installing the required dependencies, the 'echoflow' package is also automatically installed. As a result, when attempting to edit or create new flows, the system fails to recognize the changes made to the 'echoflow' package.

**Comment by Sohambutala** [comment]
For the time-being, uninstalling the echoflow manually is working. Should not be an issue when the package is deployed but needs attention while developing any components of echoflow.

**Comment by leewujung** [comment]
Did you use the `-e` (development mode) for the installation? In this mode the system would use the latest code you have modified.

**Comment by Sohambutala** [comment]
I have successfully tested using the '-e' option, and it allows me to add new flows and make modifications to existing ones. However, I am uncertain whether having a cyclic dependency is acceptable, as it can potentially introduce various issues. I looked at the 'echopype environment.cfg' file, which could be the module responsible for installing echoflow, but I couldn't find any trace of echoflow being present.
